src/plugins/data_views/common/data_views/data_view_lazy.ts

```diff
@@ -304,6 +306,8 @@ export class DataViewLazy extends AbstractDataView {
         aggregatable: true,
         searchable: true,
         count: config.popularity ?? 0,
+        customLabel: config.customLabel,
+        customDescription: config.customDescription,
```

Comment:
Were they missing here or it's expected?

Reply:
I think these are handled a couple of lines below. I think setting the count here isn't needed as its handled on line 321.